src/scripts/background.js

```diff
@@ -1714,3 +1714,16 @@ chrome.commands.onCommand.addListener(function (command) {
     }
   }
 });
+
+function handleVersionMessage (request, sender, sendResponse) {
+  if (request && request.message && request.message === 'version') {
+    sendResponse({ version: chrome.runtime.getManifest().version });
```

Comment:
Even though it should work, we might want to use `browser.runtime` here for Firefox.
Maybe store it 

```
const runtime = FF ? browser.runtime : chrome.runtime
```

and then is use so we don't have to do the check multiple times

Reply:
done